src/main/java/com/linkedin/kmf/services/MultiClusterTopicManagementService.java

```diff
@@ -296,15 +323,16 @@ void maybeAddPartitions(int minPartitionNum) {
     }
 
     void maybeReassignPartitionAndElectLeader() throws Exception {
-      KafkaZkClient zkClient = KafkaZkClient.apply(_zkConnect, JaasUtils.isZkSecurityEnabled(), ZK_SESSION_TIMEOUT_MS,
-          ZK_CONNECTION_TIMEOUT_MS, Integer.MAX_VALUE, Time.SYSTEM, METRIC_GROUP_NAME, "SessionExpireListener");
+      KafkaZkClient zkClient = KafkaZkClient.apply(_zkConnect, JaasUtils.isZkSecurityEnabled(), com.linkedin.kmf.common.Utils.ZK_SESSION_TIMEOUT_MS,
```

Comment:
I thought this was going to be using the admin client to do this.

Reply:
ReassignPartitions is not implemented in the new AdminClient. As discussed in person, instead of utilizing alterReplicaLogDirs API it might be more efficient to use the exposed reassignPartitions when it’s ready. (https://issues.apache.org/jira/browse/KAFKA-8345)